src/assets/translations/en/main.json

```diff
@@ -695,6 +695,7 @@
     "settings": {
       "settings": "Settings",
       "currency": "Currency",
+      "currencyFormat": "Currency Format",
```

Comment:
We'll need to get translations for this after this is merged. Kind of weird for a multi-currency / currency format option not to be translated, as it would logically be used by users inclined to selecting another language.

<img width="389" alt="image" src="https://user-images.githubusercontent.com/17035424/180609005-58a0b7ff-96dc-48f9-860f-e9226e9a6ffb.png">

Reply:
@gomesalexandre this will get picked up by @firebomb1 out of band as he's now owning the translations PRs coming in from gitlocalize